manifest_staging/helm/gatekeeper/values.yaml

```diff
@@ -0,0 +1,21 @@
+replicas: 1
+auditInterval: 60
+constraintViolationsLimit: 20
+auditFromCache: false
+logLevel: INFO
+image:
+  repository: quay.io/open-policy-agent/gatekeeper
+  release: v3.1.0-beta.8
+  pullPolicy: IfNotPresent
+nodeSelector: {}
+tolerations: []
```

Comment:
can we add `affinity` here too? it's defined in chart but not in values

Reply:
added